dozer/cogs/general.py

```diff
@@ -11,6 +14,104 @@ class General(Cog):
 		response = await ctx.send('Pong! We\'re in %s.' % location)
 		delay = response.created_at - ctx.message.created_at
 		await response.edit(content=response.content + '\nTook %d ms to respond.' % (delay.seconds * 1000 + delay.microseconds // 1000))
+	
+	ping.example_usage = """
+	`{prefix}ping` - Calculate and display the bot's response time
+	"""
+	
+	@command(name='help', aliases=['about'])
+	@bot_has_permissions(add_reactions=True)
+	async def base_help(self, ctx, *target):
+		"""Show this message."""
+		if not target: # No commands - general help
+			await self._help_all(ctx)
+		elif len(target) == 1: # Cog or command
+			target_name = target[0]
+			if target_name in ctx.bot.cogs:
```

Comment:
I might suggest searching for commands before cogs, as if you have a guilds cogs and a `&guilds`, the cogs documentation will be less than useful. Better yet, dynamically display both at the same time if they're named the same thing.